requirements.txt

```diff
@@ -9,6 +13,7 @@ django-mathfilters==0.3.0
 dnspython3==1.12.0
 ecdsa==0.13
 gunicorn==19.4.1
+kombu==3.0.26
 ldap3==0.9.9.3
 libsass==0.9.2
 lxml==3.5.0
```

Comment:
Instead you should add `celery[redis]` to `setup.py`'s `install_requires` list and then run `make update-requirements`.